docs_versioned_docs/version-ros2humble/ros/config/yaml/mounts.mdx

```diff
@@ -0,0 +1,178 @@
+
+#### 8x7 Riser
+<Riser8x7/>
+
+#### 1x7 Riser
+<Riser1x7/>
+
+### Custom Risers
+To promote flexibility, the size of the risers can be changed to any size; however, there are no models for these, and instead will be generated using primitives.
+
+For example, we can add a riser with a 4 by 4 grid:
+<Riser4x4/>
+
+## Bracket
+Brackets are small plates with **80mm x 80mm** screw holes that can be used to attach it to the PACS™ gird. They come in three different models.
+
+The **horizontal** and **vertical** models have screw holes to attach all small sensors such as camera, lidars, and IMU's.
+
+The **large** model have screw  holes to attach larger sensors such as the SwiftNav Duro receivers.
+
+### Horizontal Bracket
+<BracketHorizontal/>
+
+### Vertical Bracket
+<BracketVertical/>
+
+### Large Bracket
+<BracketLarge/>
+
+## Fath Pivot
+Camera mount with a single joint that can be adjusted to change the pitch of the camera. The angle of the joint is set using the **angle** parameter.
+<FathPivot/>
+
+## SICK Mounts
+These mounts are specifically designed to mount SICK LiDARs, but can be used for other sensors if required.
+The orientation of the LiDAR on the mount can be set to either `upright` or `inverted`
+
+### Upright
+<SickUpright/>
+
+### Inverted
+<SickInverted/>
+
+## Post
+Posts are vertical extrusion rails to which sensors can be added.
+
+These can be single, dual, and quad legs.
+
+### Single
+<PostSingle/>
+
+### Dual
+<PostDual/>
+
+### Quad
+<PostQuad/>
+
+## Disk
+Disk plates are circular which are use to mount circular sensors, such as the Velodyne LiDARs.
+<Disk/>
+
+We can also make use of the **post** to lift the **disk** mount.
+<DiskPost/>
+
+
+## Sample
+
+<details><summary>Sample A200 Mounts Section</summary>
+<p>
+
+<center>
+  <figure>
+    <img
+      src={require("../img/husky_accessories_0.png").default}
+      width="500"
+    />
+    <figcaption>Husky A200 with Covered User Bay</figcaption>
+  </figure>
+</center>
+
+In this sample, we will add a **bracket** to mount a LiDAR to the front of the **Husky A200**.
+
+We select the **_top_plate_mount_d1_**, i.e. the middle (**d**), front (**1**), 80mm x 80mm mounting location on the **pacs** top plate.
+
+```yaml
+bracket:
+  - parent: top_plate_mount_d1
+    xyz: [0.0, 0.0, 0.0]
+    rpy: [0.0, 0.0, 0.0]
+    model: horizontal
+```
+
+<center>
+  <figure>
+    <img
+      src={require("../img/husky_mounts_0.png").default}
+      width="500"
+    />
+    <figcaption>Husky A200 with D1 Bracket</figcaption>
+  </figure>
+</center>
+
+Then, we want to add a camera to the sensor arch. However, we will complicate things by adding it upside down to the extrusion.
+
+We choose a **fath_pivot** mount, and then we set its **parent** to the **_sensor_arch_mount_**.
+
+Using the **xyz** entry, we lower the mount by 21mm to get it under the sensor arch; then, we roll it by PI to get it upside down.
```

Comment:
- Change to _21 mm_.
- Consider changing _PI_ to _π_

Reply:
Done